src/sass/components/_footer.scss

```diff
@@ -0,0 +1,77 @@
+.footer {
+  background-color: $pageBg;
+  display: block;
+  flex-direction: column;
+  min-width: 320px;
```

Comment:
nie potrzebne

Reply:
co niepotrzebne? min-width?  display i flex direction wywalam, ale jak usuwam min-width to h2 wygląda inaczej niż na figmie